test/e2e/fixtures/wwwPageMock.js

```diff
@@ -22,7 +24,8 @@ export default function wwwPageMock(userId = null) {
 			redemptionCodeAvailableTotal: '0.00',
 		}),
 		UserAccount: () => ({
-			id: userId
+			id: userId,
```

Comment:
Based on what I've investigated, the root of the issue comes from this, `id: userId`, meaning that this problem existed before your pr, you just were unlucky enough to uncover it. It fails because when that gets serialized and sent over the wire to the mock api, `userId` doesn't get replaced with the number, it's just sent literally as `userId`. The resolver fails to read that undeclared identifier, and so it throws out the whole result. For now, I would recommend just hard-coding this as `id: 42`.

To actually set the balance in each of your tests, I recommend only using `Money: () => ...` within the cy.mock call in each test. Remove the `balance: 0` and `Money: () => '30',` from this file, because otherwise you won't be able to override the balance in each of your tests.

Reply:
Thanks for explaining that @emuvente